pkg/scheduler/factory/factory.go

```diff
@@ -516,7 +598,7 @@ func (c *Configurator) getPluginArgs() (*PluginFactoryArgs, error) {
 		StorageClassInfo:               &predicates.CachedStorageClassInfo{StorageClassLister: c.storageClassLister},
 		VolumeBinder:                   c.volumeBinder,
 		HardPodAffinitySymmetricWeight: c.hardPodAffinitySymmetricWeight,
-	}, nil
+	}, &plugins.ConfigProducerArgs{}, nil
```

Comment:
Maybe let's add this once we figure that it is needed?

Reply:
I think it is fine, we will add HardPodAffinitySymmetricWeight pretty soon.

Reply:
They can, but I don't think they should because both will be configuring very similar parameters.

Reply:
removed the err, didn't see it before!